src/Microsoft.Diagnostics.Monitoring.EventPipe/Logs/EventLogsPipeline.cs

```diff
@@ -128,11 +129,18 @@ protected override Task OnEventSourceAvailable(EventPipeEventSource eventSource,
                             args[i] = message.GetProperty(formatter.ValueNames[i]).GetString();
                         }
 
+
                         logger.Log(logLevel, new EventId(eventId, eventName), exception, formatString, args);
```

Comment:
Is this line supposed to be removed?

Reply:
Yep, I accidentally brought this back during my cleanup merges.